src/claimtrie.cpp

```diff
@@ -469,8 +524,7 @@ bool CClaimTrieCacheBase::ReadFromDisk(const CBlockIndex* tip)
 {
     LogPrintf("Loading the claim trie from disk...\n");
 
-    nNextHeight = tip ? tip->nHeight + 1 : 0;
-    base->nNextHeight = nNextHeight;
+    base->nNextHeight = nNextHeight = tip ? tip->nHeight + 1 : 0;
```

Comment:
Missing an `==`

Reply:
There is no `==` here, it assigns tip height to nNextHeight then to base one.